tests/data/fields/list_field_test.py

```diff
@@ -45,11 +48,22 @@ def test_all_fields_padded_to_max_length(self):
         numpy.testing.assert_array_almost_equal(array_dict["words"][1], numpy.array([2, 3, 4, 1, 5]))
         numpy.testing.assert_array_almost_equal(array_dict["words"][2], numpy.array([2, 3, 1, 5, 0]))
 
+    def test_nested_list_fields_are_padded_correctly(self):
+        nested_field1 = ListField([LabelField(c) for c in ['a', 'b', 'c', 'd', 'e']])
+        nested_field2 = ListField([LabelField(c) for c in ['f', 'g', 'h', 'i', 'j', 'k']])
+        list_field = ListField([nested_field1, nested_field2])
+        list_field.index(self.vocab)
+        padding_lengths = list_field.get_padding_lengths()
+        assert padding_lengths == {'num_fields': 2, 'list_num_fields': 6}
+        array = list_field.as_array(padding_lengths)
+        numpy.testing.assert_almost_equal(array, [[[0], [1], [2], [3], [4], [-1]],
```

Comment:
I think you meant to use `assert_array_equal` here?

Reply:
I actually prefer `assert_almost_equal` these days.  I think it's nicer about doing type conversions for you.